frontend/server/src/main/java/org/pytorch/serve/metrics/Metric.java

```diff
@@ -47,6 +48,8 @@ public Metric(
         this.unit = unit;
         this.hostName = hostName;
         this.dimensions = Arrays.asList(dimensions);
+        this.timestamp =
```

Comment:
what does timestamp mean exactly? the time when the the metric was instantiated or last updated? Why is this needed?